packages/headless/src/composables/tooltip/createTooltip.ts

```diff
@@ -96,16 +96,15 @@ export const createTooltip = createBuilder((options: CreateTooltipOptions) => {
     if (isOutsideClick) _isVisible.value = false;
   };
 
+  document.addEventListener("keydown", handleDocumentKeydown);
```

Comment:
I just noticed we don't have any way to clean up this global event-listener on unmount.

Reply:
We already have a onBeforeUnmount hook to clean up:
https://github.com/SchwarzIT/onyx/blob/cb3a72b4359cad361fe89f61c01fdc18d014c005/packages/headless/src/composables/tooltip/createTooltip.ts#L116